build_patch/apt/undef-bool.diff

```diff
@@ -0,0 +1,13 @@
+diff -urN a/methods/basehttp.h b/methods/basehttp.h
+--- a/methods/basehttp.h	2021-10-21 18:21:53.000000000 +0800
++++ b/methods/basehttp.h	2021-10-30 21:21:50.528112471 +0800
+@@ -20,6 +20,9 @@
+ #include <string>
+ #include <time.h>
+ 
++#undef TRUE
++#undef FALSE
```

Comment:
What's the point of this?

Reply:
to make the compile work :fr:

Reply:
Yes but what's the error without it?

Reply:
I think it is "expected identifier" in the enum following this